ibm/service/power/data_source_ibm_pi_instance_console_languages.go

```diff
@@ -6,59 +6,56 @@ package power
 import (
 	"context"
 
-	"github.com/IBM-Cloud/power-go-client/helpers"
+	"github.com/IBM-Cloud/power-go-client/clients/instance"
 	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
 	"github.com/hashicorp/go-uuid"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-
-	"github.com/IBM-Cloud/power-go-client/clients/instance"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
+// Datasource to list available console languages for an instance
 const (
-	ConsoleLanguages    = "console_languages"
-	ConsoleLanguageCode = "code"
-	ConsoleLanguageDesc = "language"
+	ConsoleLanguageCode = "Code"
 )
```

Comment:
The constants here were also used in the resource files. Eventually, I want the constants defined here gone and constants only defined in ibm_pi_constants.go. For now, to avoid having to change the data source files multiple times and also avoid compilation issues with resource files move these to ibm_pi_constants.go.

Reply:
Addressed

Reply:
@Diptipowervs  The ConsoleLanguageCode variable didn't get moved to ibm_pi_connstants.go?

Reply:
Changes made